l10n_es_aeat_sii/models/account_invoice.py

```diff
@@ -931,6 +940,13 @@ def _send_invoice_to_sii(self):
                     })
                 else:
                     inv_vals['sii_send_failed'] = True
+                if 'sii_state' in inv_vals and \
+                        not invoice.sii_account_registration_date and \
+                        'in' in invoice.type:
```

Comment:
Esto no es correcto, ya que `out_invoice` también sale True con esto. Será `invoice.type[:2] == 'in' `